tests/robustness/client/client.go

```diff
@@ -81,15 +83,13 @@ func (c *RecordingClient) Report() report.ClientReport {
 	}
 }
 
-func (c *RecordingClient) Get(ctx context.Context, key string, revision int64) (kv *mvccpb.KeyValue, rev int64, err error) {
-	resp, err := c.Range(ctx, key, "", revision, 0)
-	if err != nil {
-		return nil, 0, err
-	}
-	if len(resp.Kvs) == 1 {
-		kv = resp.Kvs[0]
-	}
-	return kv, resp.Header.Revision, nil
+func (c *RecordingClient) Do(ctx context.Context, op clientv3.Op) (clientv3.OpResponse, error) {
+	panic("not implemented")
+}
+
+func (c *RecordingClient) Get(ctx context.Context, key string, opts ...clientv3.OpOption) (*clientv3.GetResponse, error) {
+	revision := clientv3.OpGet(key, opts...).Rev()
+	return c.Range(ctx, key, "", revision, 0)
```

Comment:
I don't think it's correct.

The kubernetes interface `List` will call this function but it sets `end` as empty